dashboard/config.ru

```diff
@@ -5,8 +5,9 @@ require ::File.expand_path('../config/environment',  __FILE__)
 require 'unicorn/oob_gc'
 use Unicorn::OobGC
 use Rack::ContentLength
-if rack_env? :development
-  require 'cdo/rack/https_redirect'
-  use Rack::HTTPSRedirect
+if rack_env?(:development) && CDO.https_development
```

Comment:
yeah- it's necessary to duplicate this in development because varnish doesn't exist there. another option I considered was explicitly not supporting https at all in development environments, but that would make it more difficult to develop/debug features that interact with https in the future, so reimplementing this in the two layers keeps the environments roughly in sync.